packages/typescript/test/test.js

```diff
@@ -1181,6 +1181,32 @@ test.serial('picks up on newly included typescript files in watch mode', async (
   t.true(usage, 'should contain usage');
 });
 
+// https://github.com/rollup/plugins/issues/287. this fails with the message:
+// Unexpected token (Note that you need plugins to import files that are not JavaScript)
+test.serial.skip('works when code is in src directory', async (t) => {
```

Comment:
Last ask: Please add a note as to why this test is skipped

Reply:
sure! done